reactive/canal.py

```diff
@@ -63,7 +63,7 @@ def configure_cni():
         'etcd_key_path': ETCD_KEY_PATH,
         'etcd_cert_path': ETCD_CERT_PATH,
         'etcd_ca_path': ETCD_CA_PATH,
-        'kubeconfig_path': cni_config['kubeconfig_path'],
+        'kubeconfig_path': cni_config.get('kubeconfig_path', '/root/.kube/config'),
```

Comment:
The kubeconfig_path that kubernetes-worker sends today is `/root/cdk/kubeconfig` ([source](https://github.com/charmed-kubernetes/charm-kubernetes-worker/blob/299587028c130ed90a9f199abd73e1bafe55693d/reactive/kubernetes_worker.py#L682)). After https://github.com/juju-solutions/interface-kubernetes-cni/pull/9 lands, there will no longer be a kubeconfig_path sent from kubernetes-worker.

My main concern is that after these PRs land, we're gonna end up with old upgraded deployments using `/root/cdk/kubeconfig` (because there is a stale kubeconfig_path in the CNI config), while new deployments use `/root/.kube/config` instead. The credentials in those two files are different (kubelet vs. admin). That kind of difference between upgraded vs. new deployments can lead to bugs that are hard to diagnose.

Can you update the default here to be `/root/cdk/kubeconfig` instead?